entry_types/scrolled/package/src/editor/index.js

```diff
@@ -12,6 +12,9 @@ import * as globalInterop from 'pageflow/editor';
 
 Object.assign(pageflow, globalInterop);
 
+import i18n from 'i18n-js';
+window.I18n = i18n;
+
```

Comment:
This should not be needed since in the editor `I18n` is automatically setup to include all translations via the `i18n-js` gem.